packages/xstate-react/README.md

```diff
@@ -287,6 +287,30 @@ const App = () => {
 }
 ```
 
+## Services
```

Comment:
Note: this is what was added. The other additions were from copying it over (using `yarn docs:dev`)